docs/spec/spec-proposals/f1-fee-distribution/f1_fee_distr.tex

```diff
@@ -85,36 +105,61 @@ \subsection{Slashing}
 
 In practice this will not really be an efficiency hit, as we can expect most validators to have no slashes.
 Validators that get slashed a lot will naturally lose their delegators.
-A malicious validator that gets itself slashed many times would increase the gas to withdraw linearly, but the economic loss of funds due to the slashes should far out-weigh the extra overhead the withdrawer must pay for due to the gas.
+A malicious validator that gets itself slashed many times would increase the gas to withdraw linearly, but the economic loss of funds due to the slashes should far out-weigh the extra overhead the honest withdrawer must pay for due to the gas.
 
 \subsection{Inflation}
-Inflation is the idea that we want every staked coin to grow in value as time progresses. Each block, every staked token should each be rewarded $x$ staking tokens as inflation, where $x$ is calculated from function which takes state and the block information as input. This also allows for many seemless upgrade's to $x$'s algorithm. This can be added efficiently into the fee distribution model as follows:
+Inflation is the idea that we want every staked coin to create more staking tokens as time progresses. (To the purpose of driving down the worth of unstaked tokens) Each block, every staked token should produce $x$ staking tokens as inflation, where $x$ is calculated from a function $inflation$ which takes state and the block information as input. This inflation can be auto-bonded efficiently into the fee distribution model as follows:
+
+Make each block store the evaluation of $inflation$, which represents the number of staking tokens a single staked token should produce. 
+At the end of each period, increase the validators total number of tokens within the fee distribution by the amount due to inflation. This can be done by multiplying the validators number of staked tokens by $(1 + $ sum of x over the period$)$.
```

Comment:
This doesn't sound like it would compound (within a period). Maybe the approximation is OK in practice but it should be explicit. Could be problematic for very long periods.

Reply:
Oh this is an inconsistency which I fixed further down. It was supposed to be a product not a sum. I should have done a re-read before committing!

Reply:
:+1: